golem_messages/message.py

```diff
@@ -697,6 +697,12 @@ def __init__(self, compute_task_def=None, **kwargs):
         self.compute_task_def = compute_task_def
         super().__init__(**kwargs)
 
+    def deserialize_slot(self, key, value):
+        value = super().deserialize_slot(key, value)
+        if key == 'compute_task_def':
+            value = ComputeTaskDef(value)

```

Comment:
what happens when there is an error? is this caught and logged on a higher level?

Reply:
Yes, in `.deserialize()` https://github.com/golemfactory/golem-messages/blob/master/golem_messages/message.py#L252